src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -952,5 +955,29 @@ public async Task<string> GetNextRoatpApplicationReference()
                 return $"APR{nextInSequence}";
             }
         }
+
```

Comment:
Sorry, guys! Only the portion bellow is new. The rest is from the 'Format Document' command

Reply:
Issue addressed.